core/web-assets/src/main/assets/js/apps/onms-classifications/index.js

```diff
@@ -312,6 +314,59 @@ const confirmTopoverTemplate = require('./views/modals/popover.html');
                 });
             };
 
+            // for drag and drop of rules (redefining position)
+            $scope.sortableRules = {
+
+                start: function(e, ui) {
+                    // remember old index before moving
+                    $(ui.item).data('oldIndex', ui.item.index());
+                },
+                stop: function(e, ui) {
+
+                    // 1.) Check Precondition: not read only group
```

Comment:
Can we not numerate the comments? If we do so, if something in the process changes, all of the comments have to be updated as well, which will mostly not be done and then it is not correct anymore. I'd opt for not numerate it.

Reply:
fixed